source/index.html.md

```diff
@@ -29,148 +37,358 @@ The SMART JS client uses the open-source fhir.js for interfacing with SMART API
 * *smart.api* Non-context aware API for executing operations on all authorized resources
 * *smart.patient.api* Context aware API which automatically applies its operations to the patient in context
 
-# Register APP
-Once we have this done and hosted our APP, we will now register our APP with CERNER. Go to the link [FHIR Application Authorization Request](http://www.cerner.com/FHIR_Application_Authorization_Request/). and fill up following details
+# Registering SMART APP
+Once we have the SMART App created per the Project Setup step, get the application hosted. Your application is now ready to be registered with CERNER. Go to the link [Developer Portal APP Registration](https://developerportal.devcernerpowerchart.com/register), sign into your CERNER Care Account and fill up following details:
```

Comment:
production URL will be: https://code.cerner.com/developer/smart-on-fhir/register